Composer/packages/tools/language-servers/language-generation/src/LGServer.ts

```diff
@@ -0,0 +1,330 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as fs from 'fs';
+
+import { xhr, getErrorStatusDescription } from 'request-light';
+import URI from 'vscode-uri';
+import { IConnection, TextDocuments } from 'vscode-languageserver';
+import {
+  TextDocument,
+  Diagnostic,
+  CompletionList,
+  Hover,
+  Position,
+  CompletionItemKind,
+  CompletionItem,
+  Range,
+} from 'vscode-languageserver-types';
+import { TextDocumentPositionParams } from 'vscode-languageserver-protocol';
+import * as lg from 'botbuilder-lg';
+import get from 'lodash/get';
+
+import { buildInfunctionsMap } from './builtinFunctions';
```

Comment:
you export `buildInfunctionsMap` from a file `buildinFunctions` - they should be consistent. Perhaps export this as `functionsMap` or something a bit more expressive.

Reply:
change file name to buildInfunctionsMap.ts, add `buildIn` prefix can clarify user-defined function and lg-buildIn function.